oneview_redfish_toolkit/blueprints/event_service.py

```diff
@@ -37,11 +39,32 @@ def get_event_service():
         /redfish/v1/SessionService is requested.
 
         Returns:
-                JSON: JSON with EventService.
+            JSON: JSON with EventService.
+
+        Exceptions:
+            OneViewRedfishError: if there are invalid integers in
+                DeliveryRetryAttempts and DeliveryRetryIntervalSeconds
+                on the CONFIG file.
     """
     try:
+        try:
+            delivery_retry_attempts = \
+                int(util.config["event_service"]["DeliveryRetryAttempts"])
+            delivery_retry_interval = \
+                int(util.
+                    config["event_service"]["DeliveryRetryIntervalSeconds"])
+
+            if delivery_retry_attempts <= 0 or delivery_retry_interval <= 0:
```

Comment:
This seems the kind of thing to check during service launching... wdyt?

Reply:
I think so... fixed!